### ISSUE TITLE
view loss instead of accuracy 

### ISSUE THREAD

**Issue by jcpeterson** [revbucket/mister_ed#4]
I'd like to view the maximized crossentropy after the attack instead of accuracy, say with FGSM. How can I do that simply?

**Comment by revbucket** [comment]
Depends on how you're evaluating the result. For now, let's just say you'll have to call the loss function directly. I can add support for the EvaluationResult sometime this weekend. 

e.g. if you have a subclass of `mister_ed.loss_functions.RegularizedLoss`, you can call `forward(adversarial_tensor, labels)`. In tutorial_1, when you're displaying FGSM results, the line `attack_loss.forward(adv_examples, labels)` gets you what you want. 

(also, if your loss function were regularized by some norm on the perturbation, then the loss fxn needs to know the perturbation object so you can pass that in with the kwarg `perturbation=<perturbation_obj>`, and similarly, if you want loss per each image, pass in the kwarg `output_per_example=True` [do a `git pull` before doing the output_per_example thing though])

**Comment by jcpeterson** [comment]
What I had done before your response is forced the attack() function for the basic FGSM attack tutorial code to compute the loss again (copy and pasted) after its parameter update line and return that.

**Comment by revbucket** [comment]
Sorry for the delay in getting to this! 
Please take a look at tutorial_3 (hot off the presses) which shows how to evaluate attack loss.
